doc/GettingStarted.md

```diff
@@ -0,0 +1,225 @@
+# Getting Started
+
+On this page you will learn about [TSL](#TSL) and its Generator, what it is inteded for and how to utilize it.
+
+
+# Introduction
+
+### **What is TSL and what is it used for?**
+
+TSL is a C++ Header-only Library which abstracts elemental operations on 
+hardware specific instructions sets, espacial SIMD / vector operations.
```

Comment:
*especially?

Reply:
I would go with elementary or fundamental. Maybe also something along the lines of element-wise?

Reply:
We could also say "relevant" since primitives can be arbitrary "complex"?

Reply:
"Relevant" is a strong claim, since what's relevant to me can be irrelevant to everybody else. The overall goal is to abstract away the view on SIMD registers and the operations that you can perform on data inside them, no matter the data type of the underlying data, no?

Reply:
Agree! Well said. 

"since what's relevant to me can be irrelevant to everybody else" 
That is an interesting point which I would like to be incorporated:
We develop and add functionality to the TSL using a top-down approach: if we need a particular functionality for an algorithm, we find a suitable level of generality and add new capability to the library. Since we are IMDB people, we probably miss out on interesting features due to a lack of necessity. 
If you miss any operation relevant for your specific problem, please don't hesitate to add the functionality and contribute to the library so that everyone can benefit from! For a detailed explanation on how to contribute, see section contribute.

Reply:
I'm not sure we think in the same direction. What I wanted to say is that our main focus is on SIMD vector extension operations, but also FPGA or CUDA for example .
Nonetheless your last part sounds good.